docs/tutorials/PipelinesApi101.md

```diff
@@ -212,7 +216,7 @@ backend {
 
 **Run Workflow**
 
-`java -Dconfig.file=google.conf -jar cromwell-29.jar run hello.wdl -i hello.inputs`
+`java -Dconfig.file=google.conf -jar cromwell-67.jar run hello.wdl -i hello.inputs`
```

Comment:
The version number is already out-of-date, but perhaps this gives an indication of what version of Cromwell the documentation should work with?

Reply:
Less out of date than 29, at least... 😬 

Yeah it does almost work like a neat time capsule of when the doc was written so I'm not against hard-coding a value here... and if we're hard coding then being one version out of date is probably fine since we'll be out of date within a few weeks anyway.